mfr/extensions/codepygments/render.py

```diff
@@ -25,6 +27,9 @@ def __init__(self, *args, **kwargs):
         self.metrics.add('pygments_version', pygments.__version__)
 
     def render(self):
+        file_size = os.path.getsize(self.file_path)
+        if file_size > settings.MAX_SIZE:
+            raise(cp_exceptions.FileToLargeException('Text files larger than 64kb are not rendered; please download the file to view.'))
```

Comment:
Update: There's a library called `humanfriendly` that does this, and WaterButler's already using it:
```
In [1]: import humanfriendly

In [2]: humanfriendly.format_size??
Signature: humanfriendly.format_size(num_bytes, keep_width=False)
Source:
def format_size(num_bytes, keep_width=False):
    """
    Format a byte count as a human readable file size (supports ranges from
    kilobytes to terabytes).

    :param num_bytes: The size to format in bytes (an integer).
    :param keep_width: ``True`` if trailing zeros should not be stripped,
                       ``False`` if they can be stripped.
    :returns: The corresponding human readable file size (a string).

    Some examples:

    >>> from humanfriendly import format_size
    >>> format_size(0)
    '0 bytes'
    >>> format_size(1)
    '1 byte'
    >>> format_size(5)
    '5 bytes'
    >>> format_size(1024 ** 2)
    '1 MB'
    >>> format_size(1024 ** 3 * 4)
    '4 GB'
    """
    for unit in reversed(disk_size_units):
        if num_bytes >= unit['divider']:
            number = round_number(float(num_bytes) / unit['divider'], keep_width=keep_width)
            return pluralize(number, unit['singular'], unit['plural'])
    return pluralize(num_bytes, 'byte')
File:      ~/.virtualenvs/mss35/lib/python3.5/site-packages/humanfriendly/__init__.py
Type:      function

In [3]: humanfriendly.format_size(6553300000000000000000000000)
Out[3]: '5820499637820.83 PB'

In [4]: humanfriendly.format_size(65536)
Out[4]: '64 KB'
```